context usersharedlines: add tT options to Dial()

alexis-via opened the pull request:
I'm facing the following problem: a ring group rings, a member of the ring group answers and then we wants to transfer the call via DTMF (*2) => it doesn't work.

First, I figured out that, when calling Queue(), the options tT that allow to transfer via DTMF were not present. I added these options via a subroutine on the ring group:

```
[allowxferdtmf]
exten = s,1,NoOp(Allow transfer via DTMF)
same  =   n,Set(XIVO_GROUPOPTIONS=${XIVO_GROUPOPTIONS}Tt)
same  =   n,Return()
```

But it was still not working! I found out that, in the execution of the dialplan, after the Queue(), there was a Dial() in the context "usersharedlines" that didn't use the options tT. I added the options to this Dial() and I could eventually do the transfer via DTMF when getting the call via the ring group.

fblackburn1 commented:
I think you refer to https://wazo-dev.atlassian.net/browse/WAZO-1430

The code is still valid
https://github.com/wazo-platform/wazo-agid/blob/master/wazo_agid/handlers/groupfeatures.py#L112-L116

If you update these values on db, it should work

But we can't hardcode these values on all Dial. It's not everyone that want to allow dtmf transfer

alexis-via commented:
I reverted the changes on the code of the dialplan, removed the subroutine and executed the following SQL query:

update groupfeatures set transfer_user=1, transfer_call=1;

I'll do a test tomorrow and post the result here.

alexis-via commented:
The transfer via DTMF works. So I cancel the PR. Thank you @fblackburn1
